homeassistant/components/assist_satellite/entity.py

```diff
@@ -96,7 +96,11 @@ class AssistSatelliteAnnouncement:
     media_id: str
     """Media ID to be played."""
 
+    internal_media_id: str
```

Comment:
I've extracted this as https://github.com/home-assistant/core/pull/136141 and renamed to `original_media_id`

Reply:
Should I rebase this branch to pick up that change or clean that up later?

Reply:
rebase.